packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -439,7 +439,12 @@ export const Popup: React.FC<PopupProps> &
 
             {context.target && (
               <>
-                <EventListener listener={handleDocumentClick(getRefs)} target={context.target} type="click" capture />
+                <EventListener
+                  listener={handleDocumentClick(getRefs)}
+                  target={context.target}
```

Comment:
Such handling has an interesting side-effect as we also handle clicks for a trigger.

#### `master`

![popup-master](https://user-images.githubusercontent.com/14183168/127835649-696f1e3b-7437-4147-8c53-66e6671a0877.gif)

- click on a button to open
- mouse down on a button, move a cursor outside
- a popup is closed

#### this branch

![popup-branch](https://user-images.githubusercontent.com/14183168/127835939-f632161c-e6f7-4934-b01b-6430eb5544a6.gif)

- click on a button to open
- mouse down on a button, move a cursor outside
- a popup is **opened**

----

I am not that trigger's behavior is intentional 🤔 For the same purpose, in SUIR on mouse down we only store the event:

https://github.com/Semantic-Org/Semantic-UI-React/blob/dd4de1853068c8a0bd72095f4fda5824a0cf03f0/src/addons/Portal/Portal.js#L52-L55